### DISCUSSION TITLE
CSS

### DISCUSSION THREAD

**Discussion by AbacusQ** [reconmap/reconmap#101] [General]
I installed reconmap through docker-compose. Just wanted to edit the colors. Where can I find the CSS and or HTML files?

**Comment by santiagolizardo** [comment]
Hi @AbacusQ ; There is no easy way to change the style. Part of the CSS is defined here: https://github.com/reconmap/web-client/tree/master/src/styles but the rest comes from the Chakra UI React framework (see https://chakra-ui.com/docs/styled-system/style-props).
Thanks for posting this question.
